src/Tools/BuildBoss/CompilerNuGetCheckerUtil.cs

```diff
@@ -25,15 +25,15 @@ namespace BuildBoss
     /// The least crazy solution that could be decided on was to manage the list of dependencies 
     /// by hand in the NuSpec file and then rigorously verify the solution here.
     /// </summary>
-    internal sealed class CompilerNuGetCheckerUtil : ICheckerUtil
+    internal sealed class PackageContentsChecker : ICheckerUtil
```

Comment:
The file will be renamed to match in a later PR. Wanted to keep this easy to review.